src/v2/wallet.js

```diff
@@ -1030,21 +1030,19 @@ Wallet.prototype.prebuildTransaction = function(params, callback) {
     if (params.reqId) {
       this.bitgo._reqId = params.reqId;
     }
-    let response = yield this.bitgo.post(this.baseCoin.url('/wallet/' + this._wallet.id + '/tx/build'))
-    .send(whitelistedParams)
-    .result();
-
-    debug('postprocessing transaction prebuild: %O', response);
-
-    // postProcess prebuild may make more requests
-    if (params.reqId) {
-      response._reqId = params.reqId;
-    }
-    response = yield this.baseCoin.postProcessPrebuild(response);
-    response = _.extend({}, response, { walletId: this.id() });
-
-    debug('final transaction prebuild: %O', response);
-    return response;
+    const buildQuery = this.bitgo.post(this.baseCoin.url('/wallet/' + this._wallet.id + '/tx/build')).send(whitelistedParams).result();
```

Comment:
It's a bit easier for me to tell what's going on when the `.send()` and `.result()` are on separate lines, as it was before.